src/core/reference.js

```diff
@@ -175,7 +304,8 @@
  */
 
 /**
- * The parseInt() function parses a string argument and
+ * From <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt">the MDN entry</a>:
+ * The <a href="#/p5/parseInt">parseInt()</a> function parses a string argument and
  * returns an integer of the specified radix (the base in mathematical numeral systems).
  * @method parseInt
```

Comment:
if we include `parseInt` we should include `parseFloat`, but I'm in favor of omitting as you mentioned already.